### PR TITLE
[I18N] Update ru.js

### PR DESCRIPTION
**Pull request by Pitstopper**
Updating Russian translation.

### CODE REVIEW

File: app/i18n/locales/ru.js [hunk 1]
```diff
@@ -75,28 +76,37 @@ export default {
 	'error-user-registration-secret': 'Регистрация пользователей разрешена только через секретный URL',
 	'error-you-are-last-owner': 'Вы последний владелец. Пожалуйста, установите нового владельца, прежде чем покинуть комнату.',
 	Actions: 'Действия',
+	activity: 'активность',
+	Activity: 'Активность'
```
Comment (djorkaeffalexandre) [inline app/i18n/locales/ru.js]: Hi @Pitstopper, thanks! You forgot a comma here.. :)

File: app/i18n/locales/ru.js [hunk 2]
```diff
@@ -243,67 +333,113 @@ export default {
...
+	Show_Unread_Counter: 'Показать счетчик непрочитанных',
+	Show_Unread_Counter_Info: 'Счетчик непрочитанных отображается в виде значка справа от канала в списке каналов',
 	Sign_in_your_server: 'Войдите на ваш сервер',
 	Sign_Up: 'Регистрация',
 	Some_field_is_invalid_or_empty: 'Некоторые поля недопустимы или пусты',
-	Star_room: 'Star room',
-	Star: 'Звезда',
-	Starred_Messages: 'Помеченные сообщения',
+	Sorting_by: 'Сортировать по {{key}}',
+	Sound: 'Звук'
```
Comment (djorkaeffalexandre) [inline app/i18n/locales/ru.js]: Missing comma.

File: app/i18n/locales/ru.js [hunk 3]
```diff
@@ -243,67 +333,113 @@ export default {
 	saving_settings: 'сохранение настроек',
 	Search_Messages: 'Поиск сообщений',
 	Search: 'Поиск',
+	Search_by: 'Поиск по'
```
Comment (djorkaeffalexandre) [inline app/i18n/locales/ru.js]: Missing comma

File: app/i18n/locales/ru.js [hunk 4]
```diff
@@ -106,60 +116,109 @@ export default {
...
+	IN_APP_AND_DESKTOP: 'В приложении и на десктопе',
+	In_App_and_Desktop_Alert_info: 'Отображает баннер в верхней части экрана, когда приложение открыто, и отображает уведомление на рабочем столе.',
 	Invisible: 'Невидимый',
+	Invite: 'Приглашение',
 	is_a_valid_RocketChat_instance: 'является действительным сервером Rocket.Chat',
 	is_not_a_valid_RocketChat_instance: 'не является действительным сервером Rocket.Chat',
 	is_typing: 'печатает',
+	Invalid_server_version: 'Сервер, к которому вы пытаетесь подключиться, использует версию, которая больше не поддерживается приложением: {{currentVersion}}.\n\nНам нужна версия {{minVersion}}',
+	Join_the_community: 'Присоединиться к сообществу',
+	Join: 'Присоединиться'
```
Comment (djorkaeffalexandre) [inline app/i18n/locales/ru.js]: Missing comma

File: app/i18n/locales/ru.js [hunk 5]
```diff
@@ -106,60 +116,109 @@ export default {
...
+	Following_thread: 'Следить за темой',
 	For_your_security_you_must_enter_your_current_password_to_continue: 'В целях вашей безопасности вы должны ввести свой текущий пароль для продолжения',
 	Forgot_my_password: 'Забыл свой пароль',
 	Forgot_password_If_this_email_is_registered: 'Если эта электронная почта зарегистрирована, мы отправим инструкции о том, как сбросить пароль. Если вы не получите письмо в ближайшее время, вернитесь и повторите попытку.',
 	Forgot_password: 'Забыли пароль',
 	Forgot_Password: 'Забыли Пароль',
+	Full_table: 'Нажмите, чтобы увидеть полную таблицу',
+	Group_by_favorites: 'По избранным',
+	Group_by_type: 'По типу',
+	Hide: 'Скрыть'
```
Comment (djorkaeffalexandre) [inline app/i18n/locales/ru.js]: Missing comma

File: app/i18n/locales/ru.js [hunk 6]
```diff
@@ -243,67 +333,113 @@ export default {
...
 	User_has_been_key: 'Пользователь был {{key}}!',
 	User_is_no_longer_role_by_: '{{user}} больше не {{role}} по решению {{userBy}}',
 	User_muted_by: 'Пользователь {{userMuted}} заглушен по решению {{userBy}}',
 	User_removed_by: 'Пользователь {{userRemoved}} удален по решению {{userBy}}',
+	User_sent_an_attachment: '{{user}} отправил вложение',
 	User_unmuted_by: 'Пользователь {{userUnmuted}} перестал быть заглушенным по решению {{userBy}}',
 	User_was_set_role_by_: '{{user}} был назначен {{role}} пользователем {{userBy}}',
 	Username_is_empty: 'Имя пользователя пусто',
 	Username: 'Имя пользователя',
+	sername_or_email: 'Имя пользователя или email',
```
Comment (djorkaeffalexandre) [inline app/i18n/locales/ru.js]: `Username_or_email`

**Comment by Pitstopper** [comment]
@djorkaeffalexandre thanks for pointing out errors!

File: app/i18n/locales/ru.js (hunk 6 above)
Comment (Pitstopper) [reply app/i18n/locales/ru.js]: Necessary changes made.

**Pull request merged**